tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bottom_bar.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+
+class BottomBar(Base):
+    _remove_locator = (By.ID, 'remove')
+    _done_locator = (By.ID, 'done')
+
```

Comment:
view = self.marionette.find_element(*self._bottom_bar_locator)
position = self.marionette.find_element(*self._bottom_bar_locator).get_attribute('height')
Wait(self.marionette).until(lambda m: view.location['y'] == int(position))

This is the way I try to get the element, but not correct either...I see that the bottombar has that 'height' but is it possible to access to it? and, is this the correct approach for this comparation?